mobilecoind-json/src/bin/main.rs

```diff
@@ -203,17 +203,17 @@ fn request_code(
         .get_request_code(&req)
         .map_err(|err| format!("Failed getting request code: {}", err))?;
 
-    Ok(Json(JsonRequestCodeResponse::from(&resp)))
+    Ok(Json(JsonGetRequestCodeResponse::from(&resp)))
 }
 
-/// Retrieves the data in a request code
-#[get("/codes/request/<request_code>")]
-fn read_request(
+/// Retrieves the data in a request b58_code
+#[get("/codes/request/<b58_code>")]
+fn read_request_code(
     state: rocket::State<State>,
-    request_code: String,
-) -> Result<Json<JsonReadRequestResponse>, String> {
+    b58_code: String,
```

Comment:
Would it be better here to wrap this `b58_code` string in a `JsonReadRequestCodeRequest`? 

We're wrapping every API call in the protobuf API in a Request/Response message - so it would be more consistent to apply the same convention here.

Reply:
No, unless you want this to become a POST request. I don't think its necessary.